src/requests/cache/sorted_set/sorted_set_put_elements.rs

```diff
@@ -1,17 +1,104 @@
+use std::collections::HashMap;
+
 use momento_protos::cache_client::{SortedSetElement, SortedSetPutRequest};
 
 use crate::requests::cache::MomentoRequest;
 use crate::simple_cache_client::prep_request_with_timeout;
 use crate::{CacheClient, CollectionTtl, IntoBytes, MomentoResult};
 
+/// This trait defines an interface for converting a type into a vector of `SortedSetElement`s.
+pub trait IntoSortedSetElements: Send {
```

Comment:
I put this in this file and at the top. From what I've read the convention is to put traits before the things that consume them in the file. Let me know if this should be in its own file.

Reply:
if this is the only file where this will be consumed then this is good. if you think it might be used in any other spots then we should probably pull it out?

Reply:
I only expect it to be used externally (tests and by developers).